tests/treq.py

```diff
@@ -121,7 +121,7 @@ def size_random(self):
     def szread(self, func, sizes):
         sz = sizes()
         data = func(sz)
-        if sz >= 0 and len(data) > sz:
+        if 0 <= sz < len(data):
```

Comment:
i don't find it very readable. I would rather let it the old way.

Reply:
Okay.